### PR TITLE
Add localized entries for acex_killtracker

### PR DESCRIPTION
**Pull request by severgun**
**When merged this pull request will:**
- Title
- Add Russian translation

### CODE REVIEW

File: addons/killtracker/XEH_postInit.sqf [hunk 1]
```diff
@@ -43,15 +43,15 @@ GVAR(killCount) = 0;
     TRACE_2("kill eh",_name,_killInfo);
     // Increment kill counter
     GVAR(killCount) = GVAR(killCount) + 1;
-    GVAR(eventsArray) pushBack format ["Killed: %1 %2", _name, _killInfo];
-    GVAR(outputText) = (format ["Total Kills: %1<br/>", GVAR(killCount)]) + (GVAR(eventsArray) joinString "<br/>");
+    GVAR(eventsArray) pushBack format [LLSTRING(Killed), _name, _killInfo];
```
Comment (Dystopian) [inline addons/killtracker/XEH_postInit.sqf:46]: looks like `<br/>` should be non-localized here and below

File: addons/killtracker/XEH_postInit.sqf [hunk 2]
```diff
@@ -97,7 +97,7 @@ GVAR(killCount) = 0;
         private _unitSide = [_unit] call _fnc_getSideFromConfig;
         private _killerSide = [_killer] call _fnc_getSideFromConfig;
         if ([_unitSide, _killerSide] call BIS_fnc_areFriendly) then {
-            _killInfo pushBack "<t color='#ff0000'>Friendly Fire</t>";
+            _killInfo pushBack LLSTRING(FriendlyFire);
```
Comment (Dystopian) [inline addons/killtracker/XEH_postInit.sqf:100]: same for `<t color='#ff0000'>` and `</t>`

File: addons/killtracker/XEH_postInit.sqf (hunk 1 above)
Comment (severgun) [reply addons/killtracker/XEH_postInit.sqf:46]: I did it as it is at ACE3
https://github.com/acemod/ACE3/blob/3b15014099dc47140d0b9c8e925bab0c7a588c8a/addons/cargo/stringtable.xml#L146-L150

Comment (Dystopian) [reply addons/killtracker/XEH_postInit.sqf:46]: yeah, looks like there too

File: addons/killtracker/stringtable.xml [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+          <Original>Total Kills: %1&lt;br /&gt;</Original>
+          <Russian>Всего убийств: %1&lt;br /&gt;</Russian>
+        </Key>
+        <Key ID="STR_ACEX_KillTracker_Killed">
+          <Original>Killed: %1 %2</Original>
+          <Russian>Убил: %1 %2</Russian>
+        </Key>
+        <Key ID="STR_ACEX_KillTracker_Died">
+          <Original>Died: %1 %2</Original>
+          <Russian>Убийца: %1 %2</Russian>
```
Comment (Dystopian) [inline addons/killtracker/stringtable.xml:14]: довольно странный перевод для `Died`. Может лучше `Умер` или `Погиб`?

File: addons/killtracker/stringtable.xml [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+        </Key>
+        <Key ID="STR_ACEX_KillTracker_Vehicle">
+          <Original>Vehicle: %1</Original>
+          <Russian>Техника: %1</Russian>
+        </Key>
+        <Key ID="STR_ACEX_KillTracker_FriendlyFire">
+          <Original>&lt;t color='#ff0000'&gt;Friendly Fire&lt;/t&gt;</Original>
+          <Russian>&lt;t color='#ff0000'&gt;Союзник&lt;/t&gt;</Russian>
```
Comment (Dystopian) [inline addons/killtracker/stringtable.xml:22]: Я бы сказал `Свой` (`огонь по своим`)

File: addons/killtracker/XEH_postInit.sqf (hunk 2 above)
Comment (PabstMirror) [reply addons/killtracker/XEH_postInit.sqf:100]:
```
        <Key ID="str_a3_cfgvehicles_modulefriendlyfire_f_0">
          <Original>Friendly Fire</Original>
          <Chinese>誤擊友方</Chinese>
          <French>Tir fratricide</French>
          <Spanish>Fuego aliado</Spanish>
          <Italian>Fuoco amico</Italian>
          <Polish>Ostrzał sojuszniczy</Polish>
          <Russian>Дружественный огонь</Russian>
          <German>Teambeschuss</German>
          <Czech>Střelba do vlastních!</Czech>
          <English>Friendly Fire</English>
          <Portuguese>Fogo amigo</Portuguese>
          <Korean>아군 오인사격</Korean>
          <Chinesesimp>友军误射</Chinesesimp>
          <Japanese>同士討ち</Japanese>
          <Turkish>Dost Atışı</Turkish>
        </Key>
```

Comment (severgun) [reply addons/killtracker/XEH_postInit.sqf:100]: So? Should I remove tags or keep it consistent to other stringtables? Should I copy paste translations for this or use this ID?

Comment (PabstMirror) [reply addons/killtracker/XEH_postInit.sqf]: sorry for late reply, this is fine as people can change if the BIS wording isn't right

**Pull request merged**